Codigo/Protocol_Writes.bas

```diff
@@ -5654,7 +5654,7 @@ Public Sub WriteDebugLogResponse(ByVal UserIndex As Integer, ByVal debugType, By
         Next i
     ElseIf debugType = 1 Then
         'TODO- debug
-        Dim tIndex As Integer: tIndex = NameIndex(args(0))
+        Dim tIndex As Integer: tIndex = NameIndex(Args(0)).ArrayIndex
```

Comment:
`If IsValidUserRef(NameIndex(Args(0))) Then NameIndex(Args(0)).ArrayIndex`

Reply:
no this is fine, we need to be able to debug a user ref even if its wrong... that might be the reason why we are debugging it in the first place

Reply:
ok